index.html

```diff
@@ -37,34 +42,43 @@ <h3 class="getintouch">
             </h3>
           </div>
           <ul class="social-media-icon">
-            <li><a class="social-media" href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
-            <li> <a class="social-media" href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
-            <li><a class="social-media" href="#"><img src="./images/medium.png" alt="medium"></a> </li>
-            <li><a class="social-media" href="#"><img src="./images/github.png" alt="github"></a></li>
-            <li><a class="social-media" href="#"><img src="./images/anglelist.png" alt="anglelist"></a></li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/twitter.png" alt="twitter"></a></li>
+            <li> <a class="social-media" href="#"><img src="./images/mobile/linkedin.png" alt="linkedin"></a> </li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/medium.png" alt="medium"></a> </li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/github.png" alt="github"></a></li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/anglelist.png" alt="anglelist"></a></li>
           </ul>
         </div>
+        <div>
 
       </section>
-      <section class="section">
+      <!-- <section class="section"> -->
         <section class="second-section">
           <div class="section-2">
-            <div class="second-sec-div">
-              <div>
-                <img class="proj-img" src="images/Snapshoot Portfolio.png" alt="picture">
+            <div class="container1">
+            <div class="second-sec-div1">
+              <div >
+                <img class="proj-img" src="images//mobile/Snapshoot Portfolio.png" alt="picture">
+                <img class="proj-img1" src="images/mobile/Snapshoot Portfolio@2x.png" alt="picture">
+                
               </div>
+              <div class="B">
               <div>
                 <h2 class="proj-title">Tonic</h2>
+                <h2 class="proj-title1">Tonic</h2>
+                
               </div>
               <div class="proj-intro">
                 <span class="intro-1">CANOPY</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images//mobile/Counter.png" alt="dot">
                 <span class="intro-2">Back end Dev</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images//mobile/Counter.png" alt="dot">
                 <span class="intro-3">2015</span>
```

Comment:
- [ ] Please notice that these tags in the project cards should be on the same horizontal line, currently they are below each other please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734845-131262e4-61bf-4329-910d-7f6f5c793575.png)

figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734867-9b2eb3ba-284d-471a-a380-0c3ff2a155d7.png)

please fix these for all project cards, also the font size of these tags should be 18px currently they are 13px